src/js/utils/loader.js

```diff
@@ -7,7 +7,7 @@ const checkGenresInLocalStorage = () => {
   getMovieGenres();
 };
 
-export const loadGallery = () => {
+export const startLoader = () => {
```

Comment:
maybe better -> runLoaderWhenContentRendering runLoadingAnimationAtStartOfContentRendering

Reply:
sorry, both names you provided are too convoluted. this will stay as it is, it makes sense this way